.travis.yml

```diff
@@ -20,6 +20,9 @@ matrix:
 # These directories are cached to S3 at the end of the build
 cache:
   directories:
+    # The following two directories are for zipkin-web, which gets dependencies via NPM
+    - $HOME/.npm
+    - zipkin-web/node_modules
```

Comment:
Can we remove this line without a too big impact on build time? It's a relatively rare case that we want to remove node modules we've introduced, but still, we might want to have them removed from node_modules anyways. Some node.js tools do sanity checks on the whole node_modules folder for whatever reasons, even if some modules aren't being used.